apps/bfd-pipeline/bfd-pipeline-app/pom.xml

```diff
@@ -115,6 +115,13 @@
 			<scope>runtime</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>com.github.stefanbirkner</groupId>
+			<artifactId>system-lambda</artifactId>
+			<version>${system-lambda-version}</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
New dependency added for allowing the tests the ability to handle System.Exit() without blowing up the JVM and stopping jUnit altogether. This is test-scoped only, meaning it won't get compiled in our production server code.

Apparently stopping system.exit from killing Junit is quite annoying. Couldnt use mockito either, as the exit does not happen on the same thread as the test, which is a requirement for mocking static methods/classes.

Reply:
@StewGoin could you review this and thumbs up or down on this dependency?